environment.yml

```diff
@@ -0,0 +1,121 @@
+name: nature_news_disparities
+channels:
+  - conda-forge
+  - defaults
+dependencies:
```

Comment:
This type of file can change based on personal preference. Some (including me) only want it to contain packages that need to be installed rather than all at once. Others like to have it this way ass it guarantees that there wont be an versioning issues down the road. I won't suggest any changes to this file as I'll assume you are in the non-me category :)

Reply:
I don't have a strong opinion here, I just have experienced some r-package dependency nightmares... but I could be persuaded in either direction.

Reply:
I say leave it as I have dealt with quite a bit of conda dependency headaches myself. Better for things to work than break and have to figure it out down the road.